android/src/main/java/com/reactnativetwiliovideo/views/VideoTrackView.kt

```diff
@@ -15,10 +16,15 @@ class VideoTrackView(context: Context): LinearLayout(context) {
 
   var videoTrack: VideoTrack? = null
     set(value) {
-      if (value != field) {
-        field?.removeSink(videoView)
+      val oldValue = field
+
+      if (oldValue != value) {
+        if (oldValue != null && oldValue.sinks.contains(videoView)) {
```

Comment:
this felt bad at first, but I've come around. If we've already destroyed `oldValue`, but are hanging on to a reference here to an already released video track, it's cool to no-op on the cleanup